Gopkg.lock

```diff
@@ -240,6 +264,18 @@
   revision = "96dc06278ce32a0e9d957d590bb987c81ee66407"
   version = "v1.3.0"
 
+[[projects]]
+  name = "github.com/jinzhu/gorm"
```

Comment:
Our current entitystore implementation has no relationship model. There should be no need for an ORM library. Did you find using `lib/pq` directly insufficient? 
BTW, postgres has support for hstore and JSONB database, which might be a good fit in this use case.

Reply:
Let's discuss this a little more.  Certainly we could use JSONB or HSTORE, but we lose some of the query ability and performance of postgres.  Do we want to keep looking like a k/v store, or embrace the SQL?

Reply:
To explain my suggestion of JSONB, I was referring to the current implementation in seanhuxy/libkv, which looks like this:

```go
type KV struct {
 	Key       string `gorm:"primary_key"`
 	Value     []byte
 	UpdatedAt time.Time
 	CreatedAt time.Time
 }
```
and stores the value in `[]byte`, which is mapped to Postgres binary type `bytea` (https://github.com/jinzhu/gorm/blob/b1885a643b4977c9089d77eb07c0fd96591f94b8/dialect_postgres.go#L70)

If we use JSONB, we actually gain query ability and performance, while still using SQL:

DDL:
```sql
CREATE TABLE cards (
  id integer NOT NULL,
  board_id integer NOT NULL,
  data jsonb
);
```

Inserting:
```sql
INSERT INTO cards VALUES (3, 1, '{"name": "Cook lunch", "tags": ["Cook", "Kitchen", "Tacos"], "ingredients": ["Tortillas", "Guacamole"], "finished": false}');
```

Ability to query JSON:
```sql
SELECT * FROM cards WHERE data->>'finished' = 'true';
```

Creating indexes on JSON attributes:
```sql
CREATE INDEX idxfinished ON cards ((data->>'finished'));
```

WRT question k/v store vs SQL, it's definitely longer discussion.

Reply:
@kars7e to be honest, I haven't do too much research on different technologies regards to their performance, I will do that and we should discuss about it together tomorrow.

Reply:
Yeah, I would certainly store the values in JSONB columns rather than just text!  I think it's a reasonable step forward without getting into complex schemas and migrations.